### ISSUE TITLE
Ask about compiling openresty without luajit

### ISSUE THREAD

**Issue by XiaoFlins** [openresty/openresty#735]
Hello, I want to compile openresty from source code. If I don't have luajit, can I compile it? What needs to be modified?

**Comment by xiaocang** [comment]
Yes, just follow the openresty compilation guide, it will compile `openresty/luajit2` into it.

**Comment by houfangdong** [comment]
Thanks, according to your description, the source code of openresty cannot be compiled without luajit, right?
@xiaocang
